xtext/.gitignore

```diff
@@ -11,3 +11,4 @@ src-gen/
 xtend-gen/
 /*/target/
 /.gradle
+org.icyphy.linguafranca/.launch
```

Comment:
I guess this is better put in `xtext/org.icyphy.linguafranca/.gitignore`. I've noticed that all `.gitignore` files in the repo refer to local directories, which makes things easier to find.